### DISCUSSION TITLE
Weathermap Permission does not work for UserGroup?

### DISCUSSION THREAD

**Discussion by Squier115** [thurban/plugin_weathermap#153] [Q&A]
Hi,
i have a fresh install of Cacti 1.2.26 and installed the Weathermap-Plugin 1.3+(develop-branch).
In that Installation I have a Usergroup which should have access to a simple weathermap i created. The Users assigned to that UserGroup can navigate to the Weathermap but when they open a specific one, the image is just blank(see attachment).
Is this just me or am i doing something wrong? Can anybody help me fix this?
Thanks
![Screenshot](https://github.com/user-attachments/assets/0d2813ac-2c5b-43a0-92a5-6ccb7480b393)

**Comment by Squier115** [comment]
Someone with an Idea? I did a clean install today with cactiversion 1.2.28 and pluginversion 1.3.1 and still have the issue. As long as the usergroup sets the permission, the image appears as blank. as soon as i give the specific user from that group the permission to access the weathermap it works, but that makes the use of the groups useless.
@TheWitness: could you have a look on that please? do you need more infos from me?

Best regards

**Reply by TheWitness** [reply]
Open a new bug report please.
